package.json

```
@@ -27,5 +27,9 @@
   "engines": {
     "node": "^18.12.1"
   },
-  "packageManager": "yarn@3.3.0"
+  "packageManager": "yarn@3.3.0",
+  "dependencies": {
+    "@prisma/client": "4.10.1",
+    "prisma": "4.10.1"
+  }
```

the problem is that your adding deps to the root `package.json` (this one)
```suggestion
  "packageManager": "yarn@3.3.0"
```